editor/components/document-outline/index.js

```diff
@@ -25,6 +25,10 @@ const incorrectLevelContent = [
 	<br key="incorrect-break" />,
 	<em key="incorrect-message">{ __( '(Incorrect heading level)' ) }</em>,
 ];
+const multipleH1Headings = [
+	<br key="incorrect-break-h1" />,
+	<em key="incorrect-message-h1">{ __( '(Multiple H1 headings)' ) }</em>,
```

Comment:
Not sure about the message here. Open to suggestions.

Reply:
Seems to be missing some actionability, why would multiple H1 headings be a bad thing?

Taking inspiration from one of the posts on Yoast.com: `H1 is the title of the page or the blog post, and you should only use it once.` - See https://yoast.com/seo-basics-how-to-use-headings-on-your-site/

Reply:
@moorscode not sure I fully understand 🙂 
> why would multiple H1 headings be a bad thing?
> and you should only use it once

This PR tries to enforce that, warning users if they use it more than once.